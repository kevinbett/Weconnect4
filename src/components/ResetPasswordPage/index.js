import React from 'react';
import Navigation from '../Navigation/index';
import ResetPasswordForm from '../ResetPasswordForm/index'

const ResetPasswordPage = () => (

   <div>
       <Navigation />
       < ResetPasswordForm/>
      
   </div>
);

export default ResetPasswordPage;