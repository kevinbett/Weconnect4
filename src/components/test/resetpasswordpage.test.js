import ResetPasswordPage from '../ResetPasswordPage/index'
import { shallow } from "enzyme"

describe('Reset password page',() => { 
    it ('Reset password page renders', () => { 
        expect(shallow(<ResetPasswordPage/>)).toMatchSnapshot()
    });
} )