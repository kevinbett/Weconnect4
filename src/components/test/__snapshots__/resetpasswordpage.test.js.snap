// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reset password page Reset password page renders 1`] = `
<div>
  <Navigation />
  <ResetPassword />
</div>
`;
